base/abstractarray.jl

```diff
@@ -361,6 +382,27 @@ julia> last([1; 2; 3; 4])
 """
 last(a) = a[end]
 
+"""
+    last(v::AbstractVector, n::Integer)
+
+Get the last `n` elements of vector `v`, or fewer elements if `v` is not long enough.
+
+# Examples
+```jldoctest
+julia> last(["foo", "bar", "qux"], 2)
+2-element Array{String,1}:
+ "bar"
+ "qux"
+
+julia> last(1:6, 10)
+1:6
+
+julia> first(Float64[], 1)
```

Comment:
See typo

Reply:
Which typo?

Reply:
This is the examples section for `last`

Reply:
that the last line should be using `last` not `first`